docs/content/en/docs/concepts/cluster-topologies.md

```diff
@@ -33,6 +33,15 @@ This shows examples of a management cluster that deploys and manages multiple wo
 
 ![Management clusters can create and manage multiple workload clusters](/images/eks-a_cluster_management.png)
 
+With the management cluster in place, you have a choice of tools for creating, upgrading, and deleting workload clusters.
+Check each provider to see which tools it currently supports.
+Supported workload cluster creation tools include:
+
+* `eksctl` CLI
+* Terraform
+* GitOps
+* `kubectl` CLI or similar tools that can communicate with the Kubernetes API
```

Comment:
what do you mean similar tool here?

Reply:
I changed it to just: `kubectl` CLI to communicate with the Kubernetes API